.kitchen.yml

```diff
@@ -8,11 +8,13 @@ driver:
 
```

Comment:
I remember having that problem in the past, but I don't have it anymore...  Maybe its an upgrade issue?

```
VBox 5.1.14r112924
Chef Development Kit Version: 1.2.22
chef-client version: 12.18.31
delivery version: master (0b746cafed65a9ea1a79de3cc546e7922de9187c)
berks version: 5.6.0
kitchen version: 1.15.0
Vagrant 1.9.1
```